flw/src/main/java/org/motechproject/nms/flw/domain/ServiceUsage.java

```diff
@@ -0,0 +1,85 @@
+package org.motechproject.nms.flw.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+@Entity
+public class ServiceUsage {
+
+    public ServiceUsage(FrontLineWorker frontLineWorker, Service service,
```

Comment:
Nitpicky, but in most of our classes we define fields first, then constructors, followed by getters/setters.